.travis.yml

```diff
@@ -9,12 +9,9 @@ php:
     - hhvm
 
 matrix:
-    fast_finish: true
```

Comment:
fast finish is pointless if you don't allow anything to fail - the only thing it does is mark the build as passed if everything but the build that's allowed to fail is done